enhancements/api-review/custom-tags-aws.md

```diff
@@ -79,65 +101,373 @@ The userTags field is intended to be set at install time and is considered immut
 
 If the userTags field is changed post-install, there is no guarantee about how an in-cluster operator will respond to the change. Some operators may reconcile the change and change tags on the AWS resource. Some operators may ignore the change. However, if tags are removed from userTags, the tag will not be removed from the AWS resource.
 
+#### Existing drawbacks
+
+If a customer decides they do not want these tags applied to new resources, there is no clean way to address that need:
+
+1. they would have to get help from support to edit the status field
+2. they would have to manually remove the undesired tags from any existing resources
+
+In the future we will want to introduce a spec field where a customer can specify and edit these tags, which will be reflected into status when changed, and we will expect consuming components to reconcile changes to the set of tags by applying net new tags to existing resources (but they will still not remove tags that are dropped from the list of tags).
+
+### New proposal to support update of user-defined tags (or userTags)
+
+- `experimentalPropagateUserTags` field will be set for deprecation in this proposal version. The values in `experimentalPropagateUserTags` are ignored in this version.
```

Comment:
We cannot simply ignore the field. If the field is set to "false", then we **must not** apply the user tags to resources created in-cluster.

Reply:
Agree. Thanks.